### DISCUSSION TITLE
Post some forks

### DISCUSSION THREAD

**Discussion by jhlchu** [supermerill/SuperSlicer#3520] [General]
This branch is probably dead or dying. No active development.
Post some forks here that are worth moving to.
I'll start.
https://github.com/vovodroid2/SuperSlicer
vovodroid2 is the one who did the "Semi-parallel (range) printing", "Cool internal perimeters as external", "Round and show layer height" PRs.

**Comment by EchoAlpha519** [comment]
This one is linked in [Ellis's 3D printer tuning guide.](https://ellis3dp.com/Print-Tuning-Guide/articles/troubleshooting/small_infill_areas_overextruded.html#not-connected-top-infill-superslicer)

https://github.com/julianschill/SuperSlicer/actions 

It adds an option for minimum travel after Z lift.

**Comment by Visthum** [comment]
I hope someone can fix the arachne edition as I am getting lots of gaps In the walls and I have tried every setting and width out there. Starting to think that my extruder head ain't straight on the carrier..

**Reply by sffrankie** [reply]
The only way I was able to get rid of wall gaps was by disabling PA in klipper. Kind of a crappy trade off but my PA is typically very very low so it wasn't a huge deal. I couldn't figure out the cause until I read your comment as it's not well documented. Will have to try a print with Arachne disabled and PA on to verify but I'm pretty sure that's the cause.

**Reply by Visthum** [reply]
I switched to Prusaslicer as i am awaiting the new very big and absolute fantastic Prusaslicer 2.6 update. All my problems went away and their Arachne works flawlessly. Sure, its missing some settings like the Supporting dense layer, which i loved.. but it also provides some features that SS doesn't have. So yeah, Bye bye SS

**Comment by palumbus81** [comment]
I first moved to PrusaSlicer and was waiting for the 2.6 update; then I found a fork of the Bambu Studio (https://github.com/SoftFever/BambuStudio-SoftFever), where the features from PrusaSlicer are already included and there a very nice features available. I give them a try and it is working fine with my Creality Ender 3 V2 with Klipper.

**Reply by Visthum** [reply]
> I first moved to PrusaSlicer and was waiting for the 2.6 update; then I found a fork of the Bambu Studio (https://github.com/SoftFever/BambuStudio-SoftFever), where the features from PrusaSlicer are already included and there a very nice features available. I give them a try and it is working fine with my Creality Ender 3 V2 with Klipper.

Thanks for the tip

I've just tried it and it doesn't support marlin 2 nor overhangs with arachne... Lol. This was the thing that made me uninstall it asap..
It didn't have the features prusaslicer will get in 2.6 and further on, only snug trees. Didn't have measuring or model cutting with pin inserts female / male nor text that follows the model shape. I also found it incredibly slow on the laptop and the color scheme was horrible and that made it hard to see but I did notice one feature that I liked, when previewing it says the speed/flow/etc on the extruder coordinates.

But I'm happy you found something that fit your needs. I'm sticking with prusaslicer with arachne.
